src-docs/src/views/i18n/context.js

```diff
@@ -84,6 +89,14 @@ export default () => {
 
       <EuiSpacer size="m" />
 
+      <EuiSwitch
+        label="Use <div> other than <Fragment> to render component"
```

Comment:
Minor grammar suggestion
```suggestion
        label="Use <div> instead of <Fragment> to render component"
```